share/config.py

```diff
@@ -81,6 +83,9 @@ def __init__(
             shared_logger.warning("no namespace set in config: using `default`")
             self.namespace = "default"
 
+        if not self.tags or len(self.tags) == 0:
+            shared_logger.warning("no tags set in config")
```

Comment:
no need for a `warning`, `debug` is fine.
while we are at it we can log the `tags`, regardless they are empty or not

Reply:
removed the if statement and added: `shared_logger.debug("tags: ", extra={"tags": self.tags})` or `shared_logger.debug(f"tags: {self.tags}")`